src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/MethodologyServiceTests.cs

```diff
@@ -504,11 +710,11 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
             IMethodologyVersionRepository? methodologyVersionRepository = null,
             IMapper? mapper = null)
         {
-            return new (
+            return new(
                 contentDbContext,
                 contentPersistenceHelper ?? new PersistenceHelper<ContentDbContext>(contentDbContext),
                 mapper ?? MapperUtils.MapperForProfile<MappingProfiles>(),
-                methodologyVersionRepository ?? new Mock<IMethodologyVersionRepository>().Object
+                methodologyVersionRepository ?? Mock.Of<IMethodologyVersionRepository>(MockBehavior.Strict)
```

Comment:
If you ever work in Tests.cs files with the old way of providing default Mocks (`new Mock<Blah>().Object`), this regex search and replace might be handy in Rider:

Search for: `new Mock<(.*)>\(\).Object`
Replace with: `Mock.Of<$1>()`

Reply:
That's really cool, thanks for letting me know about that. 👍